akamai: add troubleshooting section (#375)
flatcar/flatcar-website: content/docs/latest/installing/community-platforms/akamai.md

Check out the [Flatcar Container Linux Quickstart][quickstart] guide, or dig
into [more specific topics][doc-index].

## Troubleshooting and known issues

* If the instance is booted with a Private IP option, the IP will not be set automatically on Flatcar. It is the "Linode helpers" that take care of this, to do so on Flatcar: `ip addr add "${COREOS_AKAMAI_PRIVATE_IPV4_0}" dev eth0`

> * Is it not possible to get SSH keys stored on the user account with Afterburn, if SSH keys are needed on the instance it will be through Ignition. (See: https://github.com/coreos/afterburn/issues/1111 and https://www.linode.com/community/questions/24938/v1ssh-keys-is-empty-when-deploying-a-private-image)

Reviewer: ```suggestion
* It is not possible to configure SSH keys against a user account with Afterburn. If SSH keys are needed on the instance, it must be done through Ignition. (See: https://github.com/coreos/afterburn/issues/1111 and https://www.linode.com/community/questions/24938/v1ssh-keys-is-empty-when-deploying-a-private-image)
```
